scripts/termux-bluetooth-scan.in

```diff
@@ -0,0 +1,25 @@
+#!@TERMUX_PREFIX@/bin/sh
+set -e -u
+
+SCRIPTNAME=termux-bluetooth-scan
+show_usage () {
+    echo "Usage: $SCRIPTNAME <start/stop/info>"
```

Comment:
I requested a change to `info` in https://github.com/termux/termux-api/pull/686 - update this once we agree on a verb over there.